src/aks-preview/azext_aks_preview/managed_cluster_decorator.py

```diff
@@ -4060,11 +4069,27 @@ def update_enable_advanced_network_observability_in_network_profile(self, mc: Ma
 
         advanced_network_observability = self.context.get_enable_advanced_network_observability()
         if advanced_network_observability is not None:
-            mc.network_profile.advanced_networking = self.models.AdvancedNetworking(  # pylint: disable=no-member
-                observability=self.models.AdvancedNetworkingObservability(  # pylint: disable=no-member
-                    enabled=advanced_network_observability
-                )
-            )
+            if mc.network_profile.advanced_networking is None:
```

Comment:
adding so that updating the observability enablement does not affect other properties within the advanced networking profile.